extras/anthos-vmruntime/README.md

```diff
@@ -54,11 +54,13 @@ Environment=SPRING_PROFILES_ACTIVE=inmemory
     ```
 
 - Make the bucket **publicly readable**
+    > **Note**: For typical use the bucket should be **private**, and [credentials](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/vm-runtime/create-storage-credentials) added in VM Runtime to securely access the private image.
```

Comment:
**(Nit)** 🙏🏼 

.... and credentials **for the bucket** added **into** VM Runtime to securely access the private image.

Reply:
SG